src/app/containers/CpsRecommendations/utils.js

```diff
@@ -1,3 +1,11 @@
+const getIchefSrc = href => {
+  const DEFAULT_IMAGE_RES = 660;
+
+  const path = href.replace('http://c.files.bbci.co.uk', '/cpsprodpb');
+
+  return `https://ichef.bbci.co.uk/news/${DEFAULT_IMAGE_RES}${path}`;
```

Comment:
Actually do we need to do this? It looks like `StoryPromo` might do this conversion for us https://github.com/bbc/simorgh/blob/93007af14b99f476605d3b8888e5127fd664b3d5/src/app/containers/StoryPromo/index.jsx#L46

Reply:
:scream:

Reply:
thanks @HarveyPeachey I was previously just using the `StoryPromo` from psaammead which meant  had to do tthe transformation myself but it's been switched to use this now.